NEWS.md

```diff
@@ -28,6 +29,53 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+### Added
+
+- Add `onRelease` option to `<keybind>` in support of binding `Super_L` to a
+  menu. Written-by: @spl237 PR #1888
+- Add support for `security-context-v1`. Written-by: @nesteroff PR #1817
+- Add partial support for `tablet-v2-manager`. Written-by: @jp7677 PR #1678
+- Add action `UnMaximize`. PR #1831
+- Support multiple IME popups. PR #1823
+- Add `All` context for mouse bindings which need to be handled irrespective of
+  the mouse pointer location. This enables Super+mouse-scroll to change
+  magnification. Written-by: @spl237 PR #1768
+- Add `SetDecorations` action. Written-by: @xi PR #1733
+- Add `policy` option to `AutoPlace` action. PR #1784
+- Add window type filter to If-actions. Written-by: @xi PR #1731
+- Add screen magnifier which can be controlled with the `ZoomIn`, `ZoomOut` and
+  `ToggleMagnify` actions. Written-by: @spl237 PR #1774
+
+### Fixed
+
+- Prevent child views from opening outside of usable area. PR #1878
+- Fix IME popups issues (flicker when popup surface is initially mapped
+  and bug which incorrectly shows multiple popups). PR #1872
+- Rate-limit cursor-driven resize events based on monitor's refresh rate. This
+  fixes the lag when resizing windows of some apps on XWayland, for example
+  Chromium and Steam. PR #1861
+- Session-lock: fix flashing & update cursor shape. PR #1858
+- Remove tearing-controller listeners on destroy. PR #1853
+- Fix action parsing bug. PR #1838
```

Comment:
Suggest something like "Handle invalid `ForEach` and `If` action configs" instead.

Reply:
Done